src/test/ui/repr/repr-align.stderr

```diff
@@ -1,74 +1,74 @@
-error[E0589]: invalid `repr(align)` attribute: not an unsuffixed integer
-  --> $DIR/repr-align.rs:3:8
+error[E0589]: invalid `repr(align)` attribute
```

Comment:
Is it okay to lose the additional info in the primary message?

Reply:
As long as it is also present in the label, as it is here.